x/committee/msg_server_test.go

```diff
@@ -0,0 +1,208 @@
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	proposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/committee"
+	"github.com/kava-labs/kava/x/committee/keeper"
+	"github.com/kava-labs/kava/x/committee/types"
+	swaptypes "github.com/kava-labs/kava/x/swap/types"
+)
+
+//var _ types.PubProposal = UnregisteredPubProposal{}
+//
+//// UnregisteredPubProposal is a pubproposal type that is not registered on the amino codec.
+//type UnregisteredPubProposal struct{}
+//
+//func (UnregisteredPubProposal) GetTitle() string       { return "unregistered" }
+//func (UnregisteredPubProposal) GetDescription() string { return "unregistered" }
+//func (UnregisteredPubProposal) ProposalRoute() string  { return "unregistered" }
+//func (UnregisteredPubProposal) ProposalType() string   { return "unregistered" }
+//func (UnregisteredPubProposal) ValidateBasic() error   { return nil }
+//func (UnregisteredPubProposal) String() string         { return "unregistered" }
+
+// NewDistributionGenesisWithPool creates a default distribution genesis state with some coins in the community pool.
+//func NewDistributionGenesisWithPool(communityPoolCoins sdk.Coins) app.GenesisState {
+//gs := distribution.DefaultGenesisState()
+//gs.FeePool = distribution.FeePool{CommunityPool: sdk.NewDecCoinsFromCoins(communityPoolCoins...)}
+//return app.GenesisState{distribution.ModuleName: distribution.ModuleCdc.MustMarshalJSON(gs)}
+//}
+
+type MsgServerTestSuite struct {
+	suite.Suite
+
+	app       app.TestApp
+	keeper    keeper.Keeper
+	msgServer types.MsgServer
+	ctx       sdk.Context
+	addresses []sdk.AccAddress
+
+	communityPoolAmt sdk.Coins
+}
+
+func (suite *MsgServerTestSuite) SetupTest() {
+	_, suite.addresses = app.GeneratePrivKeyAddressPairs(5)
+	suite.app = app.NewTestApp()
+	suite.keeper = suite.app.GetCommitteeKeeper()
+	suite.msgServer = committee.NewMsgServerImpl(suite.keeper)
+	encodingCfg := app.MakeEncodingConfig()
+	cdc := encodingCfg.Marshaler
+
+	memberCommittee, err := types.NewMemberCommittee(
+		1,
+		"This committee is for testing.",
+		suite.addresses[:3],
+		[]types.Permission{&types.GodPermission{}},
+		sdk.MustNewDecFromStr("0.5"),
+		time.Hour*24*7,
+		types.TALLY_OPTION_FIRST_PAST_THE_POST,
+	)
+	suite.Require().NoError(err)
+
+	firstBlockTime := time.Date(1998, time.January, 1, 1, 0, 0, 0, time.UTC)
+	testGenesis := types.NewGenesisState(
+		3,
+		[]types.Committee{memberCommittee},
+		[]types.Proposal{},
+		[]types.Vote{},
+	)
+	suite.communityPoolAmt = sdk.NewCoins(sdk.NewCoin("ukava", sdk.NewInt(1000)))
+	suite.app.InitializeFromGenesisStates(
+		app.GenesisState{types.ModuleName: cdc.MustMarshalJSON(testGenesis)},
+		// TODO: not used?
+		//NewDistributionGenesisWithPool(suite.communityPoolAmt),
```

Comment:
This wasn't required to pass the tests.  I'm assuming it was in the old handler tests with the intention to test the community spend proposal